src/Divider/Divider.tsx

```diff
@@ -9,12 +9,14 @@ export type DividerDefaultProps = {
   size: Divider.Size,
   /** an optional style object to pass to top level element of the component */
   style: React.CSSProperties,
+  /** an optional class name to pass to first inner element of the component */
+  className?: Divider.className;
 };
 
 export namespace Divider {
   export type Orientation = 'horizontal' | 'vertical'
   export type Size = 'small' | 'medium' | 'large' | 'no-margin'
-
+  export type className = string;
```

Comment:
we're creating a type alias for the the other two because they are complex types, but for a simple string is not necessary.

please remove this and replace `Divider.className`  above with `string`